LayoutTests/fast/block/positioning/fixed-position-transform-related-container-expected.html

```diff
@@ -0,0 +1,6 @@
+<div style="position:absolute; left:50px; transform:translateX(0)">
```

Comment:
A ref test that is identical to the test doesn't make much sense. For "should not assert" tests, it should be a text test (`testRunner.dumpAsText()`) which some output that says "This test should not assert in debug". @aprotyas confused you!

Reply:
My bad, apologies for the confusion.

Reply:
@aprotyas NP! Thanks for reviewing actually.